blocks/api/post.pegjs

```diff
@@ -296,12 +352,19 @@ Block_Name_Part
   = $( [a-z][a-z0-9_-]* )
 
 Block_Attributes
-  = attrs:$("{" (!("}" WS+ """/"? "-->") .)* "}")
+  = attrs:$("{" (!("}") .)* "}")
   {
     /** <?php return json_decode( $attrs, true ); ?> **/
     return maybeJSON( attrs );
   }
 
+Block_Version
+  = "v" v:$( [0-9]* )
+  {
+    /** <?php return (int) $v; ?> **/
+    return parseInt( v, 10 );
+  }
```

Comment:
It looks like we're wanting to see something like `v153840` here - is there a reason not to rely instead on an HTML-like attribute? seems like that could be less surprising to someone seeing it

```html
<!-- wp:block v="1358" /-->
```

With `v1348` it looks like we have an attributes whose name is `v1348` and whose value is `true`

Reply:
No reason and preference here